integration_test/gce/gce_testing.go

```diff
@@ -415,14 +415,15 @@ func lookupTrace(ctx context.Context, logger *log.Logger, vm *VM, window time.Du
 // time series. An error is returned if the evaluation fails.
 // A return value of (nil, nil) indicates that the evaluation succeeded
 // but returned no data.
-func nonEmptySeries(logger *log.Logger, it *monitoring.TimeSeriesIterator) (*monitoringpb.TimeSeries, error) {
+func nonEmptySeries(logger *log.Logger, it *monitoring.TimeSeriesIterator) ([]*monitoringpb.TimeSeries, error) {
```

Comment:
* Please update the function name and header so it matches the new behaviour
* This will no longer terminate early in case the caller only needs a single time series. I would add a `minimumRequiredSeries int` parameter to facilitate that.